secure_strings.tf

```
@@ -0,0 +1,22 @@
+locals {
+  secure_strings_count = length(keys(var.secure_strings)) > 0 ? 1 : 0
+  secure_strings_exports = local.secure_strings_count > 0 ? join(
+    "\n",
+    [
+      for key, _ in var.secure_strings : "export ${key}=$(aws secretsmanager get-secret-value --secret-id ${local.base_name}-secret --query SecretString --output text | jq -r '.${key}')"
+    ]
+  ) : ""
```

Maybe something like?
```suggestion
  secure_strings_exports = local.secure_strings_count > 0 ? join(
    "\n",
    "SECRETVALUE=$(aws secretsmanager get-secret-value --secret-id ${local.base_name}-secret --query SecretString --output text)"
    [
      for key, _ in var.secure_strings : "export ${key}=$(echo $SECRETVALUE | jq -r '.${key}')"
    ]
  ) : ""
```

---

Ah, good idea, then we only grab the secret once. Ill make this change 👍